*** CMakeLists.txt	2020-01-10 12:10:39.119057615 +0100
--- CMakeLists.txt.new	2020-01-10 12:04:14.727498370 +0100
***************
*** 19,29 ****
  set (wxDATABASE_VERSION_MINOR 0)
   
  #Set Options 
! option (ENABLE_MYSQL "Enable MariaDB and MYSQL Database" OFF) 
! option (ENABLE_SQLITE "Enable SQLite3 Database" OFF) 
  option (ENABLE_PGS "Enable PostGreSQL Database" OFF) 
! option (ENABLE_TDS "Enable MSSQL Access with TDS Database" ON) 
! option (ENABLE_ODBC "Enable ODBC Access" ON) 
  option (BUILD_SAMPLE "Build Sample Application" ON) 
  #Build type
  #set(CMAKE_BUILD_TYPE Debug)
--- 19,30 ----
  set (wxDATABASE_VERSION_MINOR 0)
   
  #Set Options 
! option (ENABLE_MYSQL "Enable MariaDB and MYSQL Database" ON) 
! set(MYSQL_LIB_DIR ${MYSQL_LIB_DIR} "/usr/lib/x86_64-linux-gnu/")
! option (ENABLE_SQLITE "Enable SQLite3 Database" ON) 
  option (ENABLE_PGS "Enable PostGreSQL Database" OFF) 
! option (ENABLE_TDS "Enable MSSQL Access with TDS Database" OFF) 
! option (ENABLE_ODBC "Enable ODBC Access" OFF) 
  option (BUILD_SAMPLE "Build Sample Application" ON) 
  #Build type
  #set(CMAKE_BUILD_TYPE Debug)
*** samples/wxDatabaseApp.cpp	2020-01-10 12:22:05.873802748 +0100
--- samples/wxDatabaseApp.cpp.new	2020-01-10 12:20:14.900708580 +0100
***************
*** 40,46 ****
  		// simply comment in one of the GetDatabase() calls below
  
  		// use the wxDatabaseApp.conf file
! 		//pDatabase = GetDatabase();
  		
  		// SQLite3 direct
  		//pDatabase = GetDatabase("[SQLite]\ndatabase=C:/wxDev/sqlite/test.sqlite");
--- 40,46 ----
  		// simply comment in one of the GetDatabase() calls below
  
  		// use the wxDatabaseApp.conf file
! 		pDatabase = GetDatabase();
  		
  		// SQLite3 direct
  		//pDatabase = GetDatabase("[SQLite]\ndatabase=C:/wxDev/sqlite/test.sqlite");
***************
*** 61,67 ****
  //	instance = sqlexpress
  //#	port = 1433
  // instance and port are mutually exclusive
! 		pDatabase = GetDatabase("[TDS]\nfreetds=C:/wxDev/freetds-1.00.24/freetds.conf\nserver=manyleaves_sqlexpress\ndatabase=test\nuser=\npassword=\nversion=7.1");
  	
  		// TDS via ODBC
  		//pDatabase = GetDatabase("[ODBC]\nConnection=DRIVER=SQL Server;SERVER=manyleaves\\\\sqlexpress;TRUSTED_CONNECTION=Yes;DATABASE=test\nDSN=\nDbType=TDS\n");
--- 61,67 ----
  //	instance = sqlexpress
  //#	port = 1433
  // instance and port are mutually exclusive
! 		//pDatabase = GetDatabase("[TDS]\nfreetds=C:/wxDev/freetds-1.00.24/freetds.conf\nserver=manyleaves_sqlexpress\ndatabase=test\nuser=\npassword=\nversion=7.1");
  	
  		// TDS via ODBC
  		//pDatabase = GetDatabase("[ODBC]\nConnection=DRIVER=SQL Server;SERVER=manyleaves\\\\sqlexpress;TRUSTED_CONNECTION=Yes;DATABASE=test\nDSN=\nDbType=TDS\n");
